### PR TITLE
05_search test cases added

### PR DESCRIPTION
**Pull request by korkoza**

### CODE REVIEW

File: pages/searchPage.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+        return new BaseElement(element(by.css(".products-list.cell-list")), "Result");
+    }
+
+    getSearchResult() {
+        return new TextViewElement(element(by.css("#page-search h1")), "Search result");
+    }
+
+    getFirstItem() {
+        return new TextViewElement(element(by
+            .css(".products-list.cell-list>li:nth-child(1)>div:nth-child(3)>p")), "First search item");
```
Comment (nbabii) [inline pages/searchPage.js]: Very complex selector, try to optimize.

Comment (korkoza) [reply pages/searchPage.js]: changed to "li:nth-child(1)>div.item-info>p"

Comment (nbabii) [reply pages/searchPage.js]: good, but please check this one `.product-item:nth-child(1) .h4`

Comment (korkoza) [reply pages/searchPage.js]: yes, it looks better. thanks for suggestion.
fixed

**Pull request merged**